packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -3,6 +3,8 @@ Table
 
 Tables show tabular data in columns and rows.
 
+@status Beta
```

Comment:
I think we should actually be using Draft here, you can see this in `PageHeader`. I'm also going to update this status stuff in a different ticket: https://jira.cms.gov/browse/WNMGDS-532